draft-ietf-lake-reqs.md

```diff
@@ -168,13 +169,11 @@ Assuming that both signature public keys and static DH public keys are in use, t
 
 ## Mutual Authentication {#mutual-auth}
 
-The AKE must provide mutual authentication during the protocol run. At the end of the AKE protocol, each endpoint shall have authenticated the other.
-
-The AKE cannot rely on messages being exchanged in both directions after the AKE has completed, because CoAP/OSCORE requests may not have a response {{RFC7967}}. Furthermore, there is no assumption of dependence between CoAP client/server and AKE initiator/responder roles, and an OSCORE context may be used with CoAP client and server roles interchanged as is done e.g. in {{LwM2M}}. Since the protocol may be initiated by different endpoints, it shall not be necessary to determine beforehand which endpoint takes the role of initiator of the AKE.
```

Comment:
This text seems misplaced in the mutual authentication section. Relocate? For now, I removed it, which is too aggressive, but would be better to move this where it is needed.

Reply:
I propose moving the relevant text to section 2.1 which is detailing the specifics of an AKE for OSCORE. I made an issue to fix after merge.

https://github.com/lake-wg/reqs/issues/16